src/constants/validations.js

```diff
@@ -181,6 +181,24 @@ exports.isValidMember = (member) => {
   return true;
 };
 
+/**
+ * Validation of subscriber on submission or update.
+ * @param {object} subscriber - Subscriber information to be validated.
+ * @returns {boolean} True if valid. False with error message if invalid.
+ */
+exports.isValidSubscriber = (subscriber) => {
+  if (!ifExists(subscriber.firstname, "Enter the subscriber's first name."))
+    return false;
+  if (!ifExists(subscriber.lastname, "Enter the subscriber's last name."))
+    return false;
+
+  const isNotSusbscribed = subscriber.subscription !== false;
+  if (!ifExists(isNotSusbscribed, 'Check box for subscribed.')) return false;
+  if (!ifExists(subscriber.createtime, "Select the subscriber's create time."))
+    return false;
+  return true;
+};
+
```

Comment:
This isn't necessary

Reply:
This shouldnt be in this commit because it was part of the add commit